components/Code/index.tsx

```diff
@@ -27,74 +26,39 @@ function CopyCode({ codeRef, rootClass = 'rdmd-code-copy', className = '' }) {
       setTimeout(() => $el.classList.remove(copyClass), 1500);
     }
   };
+
   return <button ref={button} aria-label="Copy Code" className={`${rootClass} ${className}`} onClick={copier} />;
 }
 
-CopyCode.propTypes = {
-  className: string,
-  codeRef: oneOfType([func, shape({ current: instanceOf(Element) })]).isRequired,
-  rootClass: string,
-};
-
 function Code(props) {
-  const { children, className, copyButtons, lang, meta, theme } = props;
+  const { children, copyButtons, lang, meta, theme, value } = props;
 
-  const langClass = className.search(/lang(?:uage)?-\w+/) >= 0 ? className.match(/\s?lang(?:uage)?-(\w+)/)[1] : '';
-  const language = canonicalLanguage(lang) || langClass;
+  const language = canonicalLanguage(lang);
 
-  const codeRef = createRef();
+  const codeRef = createRef<HTMLElement>();
 
   const codeOpts = {
     inline: !lang,
     tokenizeVariables: true,
     dark: theme === 'dark',
   };
 
-  const codeContent =
-    syntaxHighlighter && children ? syntaxHighlighter(children[0], language, codeOpts) : children?.[0] || '';
+  const code = value ?? children?.[0] ?? children ?? '';
+  const highlightedCode = syntaxHighlighter && code ? syntaxHighlighter(code, language, codeOpts) : code;
 
   return (
-    <Fragment>
+    <>
       {copyButtons && <CopyCode className="fa" codeRef={codeRef} />}
       <code
         ref={codeRef}
         className={['rdmd-code', `lang-${language}`, `theme-${theme}`].join(' ')}
         data-lang={language}
-        name={meta}
         suppressHydrationWarning={true}
       >
-        {codeContent}
+        {highlightedCode}
       </code>
-    </Fragment>
+    </>
   );
 }
 
-function CreateCode({ copyButtons, theme }) {
-  // eslint-disable-next-line react/display-name
-  return props => <Code {...props} copyButtons={copyButtons} theme={theme} />;
-}
-
-Code.propTypes = {
-  children: arrayOf(string),
-  className: string,
-  copyButtons: bool,
-  lang: string,
-  meta: string,
-  theme: string,
-};
-
-Code.defaultProps = {
-  className: '',
-  copyButtons: true,
-  lang: '',
-  meta: '',
-};
-
-CreateCode.sanitize = sanitizeSchema => {
-  // This is for code blocks class name
-  sanitizeSchema.attributes.code = ['className', 'lang', 'meta', 'value'];
-
-  return sanitizeSchema;
-};
```

Comment:
oohh do we not need this sanitize step anymore

Reply:
Pretty sure. It was to prevent code execution, and we're kind of throwing that out the window.